index.html

```diff
@@ -84,8 +142,9 @@
                 let variants = []
                 let rolls = 1
                 let fails = 0
-                let max = allVariants.length
+                let max = oneShots.length + permuteThese.length
```

Comment:
We should just leave this to be based on `allVariants`, you already push all the permutations on 105-108.

Reply:
The only issue with that is that then anything with a permutation is 24 times more likely to appear than anything that isn't. If we're fine with that, swap it to allVariants. But I'm personally under the impression that we want the non permutated callouts to appear roughly at the same rate as the permuted ones.